libs/agno/agno/agent/agent.py

```diff
@@ -4173,32 +4246,24 @@ async def aprint_response(
                     panels.append(thinking_panel)
                     live_log.update(Group(*panels))
 
-                response_content_batch: Union[str, JSON, Markdown] = ""
```

Comment:
We still need some of this formatting, no?

Reply:
hmh, have to refactor the `aprint_response` function, checking